brambling/views/organizer.py

```diff
@@ -1298,11 +1298,13 @@ def get_forms(self):
                         break
         self.transaction_refund_forms = [TransactionRefundForm(t)
                                          for t in self.order.transactions.all()]
+
+        # Transaction refund forms are not handled by this view so don't
+        # get included here.
+        forms = [self.notes_form] + self.attendee_forms
         if can_edit_event:
-            forms = [self.payment_form, self.notes_form, self.transaction_refund_forms]
-        else:
-            forms = [self.notes_form]
-        return forms + self.attendee_forms
+            forms.append(self.payment_form)
```

Comment:
Transaction refund forms still need to be generated - they're used by get_context_data - but they are not one of the forms that needs to be processed on POST.

Basically, `get_forms()` generates all the forms, whether or not they're supposed to be displayed. The return value just determines whether the form will be processed on POST.